.gruntjshintrc

```diff
@@ -0,0 +1,18 @@
+{
+    "boss": true,
```

Comment:
Since in JavaScript there is no real difference between singles and doubles I decided to do everything with doubles, so if there is singlequote in a textstring it doesn't have to be escaped.